.github/workflows/build_test_linux.yml

```diff
@@ -2,53 +2,53 @@ name: build_test_linux
 on:
   push:
     branches:
-    - main
+      - main
```

Comment:
Why got all files formatted? Do you use black?

Reply:
Can you double-check? I think we should add a GitHub Action step to validate code formatting, otherwise we run into the same issue again

Reply:
Ah thanks!